dashboard/src/modules/components/OverviewComponent/common-component.jsx

```diff
@@ -157,3 +171,164 @@ export const RenderPagination = (props) => {
     />
   );
 };
+
+export const EditRow = (props) => {
+  return (
+    <div className="pf-c-inline-edit__action pf-m-enable-editable">
+      {!props.item.isEdit ? (
+        <Button
+          variant="plain"
+          onClick={() => props.toggleEdit(props.item.resource_id, true)}
+          icon={<PencilAltIcon />}
+        />
+      ) : (
+        <div>
+          <Button
+            isDisabled={
+              !props.item.isDirty ||
```

Comment:
Can we instead reference this as
```
              !props.item["isDirty"] ||
```
If so, that would enable us to replace the `"isDirty"` with a symbolic constant, which would make the code more consistent/robust.

Analogously, `isEdit` at line 178 and probably others.

Reply:
Updated